### PR TITLE
Add print commands for Hanyin HM-A300L

### PR DESCRIPTION
**Pull request by luiges90**
#### What does this change?
Add following commands to HM-A300L:
- Set paper type
- Set print area size
- Add text
- Set bold
- Set text size
- Get printer status 

#### Checklist
- [x] Run the `pre_pr.sh` script to ensure code quality.

### CODE REVIEW

File: android/src/main/kotlin/hk/gogovan/flutter_label_printer/FlutterLabelPrinterPlugin.kt [hunk 1]
```diff
@@ -30,11 +30,11 @@ class FlutterLabelPrinterPlugin : FlutterPlugin, ActivityAware {
         methodHandler = FlutterLabelPrinterMethodHandler(context, bluetoothSearcher)
 
         channel =
-            MethodChannel(flutterPluginBinding.binaryMessenger, "com.gogovan/flutter_label_printer")
+            MethodChannel(flutterPluginBinding.binaryMessenger, "hk.gogovan.label_printer.flutter_label_printer")
```
Comment (gogovan-jacksonchung) [inline android/src/main/kotlin/hk/gogovan/flutter_label_printer/FlutterLabelPrinterPlugin.kt:33]: the app bundle need to be `hk.gogovan.label_printer` in order to use this plugin? 🤔 correct me if I'm wrong

Comment (luiges90) [reply android/src/main/kotlin/hk/gogovan/flutter_label_printer/FlutterLabelPrinterPlugin.kt:33]: no, this is just for namespacing purposes. We can use any string for this.

File: pre_pr.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ set -e
 set -x
 
 flutter format lib
-flutter format test
+# flutter format test
```
Comment (gogovan-jacksonchung) [inline pre_pr.sh:8]: why no need to format test files?

File: android/src/main/kotlin/hk/gogovan/flutter_label_printer/FlutterLabelPrinterPlugin.kt (hunk 1 above)
Comment (gogovan-jacksonchung) [reply android/src/main/kotlin/hk/gogovan/flutter_label_printer/FlutterLabelPrinterPlugin.kt:33]: got it!

File: pre_pr.sh (hunk 2 above)
Comment (luiges90) [reply pre_pr.sh:8]: there are no `test` file in the first place 🙈 .
If tests are added, uncomment this line.

**Comment by gogovan-jacksonchung** [comment]
the code looks fine. but the CI has failed @luiges90

File: pre_pr.sh (hunk 2 above)
Comment (gogovan-jacksonchung) [reply pre_pr.sh:8]: okok

**Comment by luiges90** [comment]
> the code looks fine. but the CI has failed @luiges90

updated, check again

**Review by gogovan-jacksonchung — APPROVED** [review]

**Pull request merged**